source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -122,6 +130,54 @@ example outputs the following text:
    resumeToken={...},
    ...
 
+.. _kotlin-change-stream-split-large-event:
+
+Split Large Change Stream Events
+--------------------------------
+
+When connecting to a MongoDB deployment running version 7.0 or later,
+you can use the ``$changeStreamSplitLargeEvent`` aggregation operator to
+split event documents that exceed 16 MB into smaller fragments.
+
+Use the ``$changeStreamSplitLargeEvent`` operator only when necessary. For
```

Comment:
Suggestion:

I think it could be helpful to identify the specific condition in this sentence, even though I think I wrote the same thing in the Java docs.

```suggestion
Use the ``$changeStreamSplitLargeEvent`` operator only when you expect
the change stream events to exceed the 16 MB document size limit. For
```